### PR TITLE
doc: add more doc on pull-request and Pastamaker

### LINKED ISSUE #766: Document pastamaker usage
We should add a note about Pastamaker usage in the doc.

### PR DESCRIPTION
**Pull request by jd**
Closes #766

### CODE REVIEW

File: doc/source/contributing.rst [hunk 1]
```diff
@@ -33,7 +33,30 @@ When opening a pull-request, make sure that:
   `git rebase --interactive` and/or `git commit --amend`.
 * We recommend using `git pull-request`_ to send your pull-requests.
 
+All sent pull-requests are checked using `Travis-CI`_, which is in charge of
+running the tests suites. They are different scenarios being run: `PEP 8`_
```
Comment (chungg) [inline doc/source/contributing.rst]: s/They/There/

File: doc/source/contributing.rst [hunk 2]
```diff
@@ -33,7 +33,30 @@ When opening a pull-request, make sure that:
...
+compliance tests, ugrade tests, unit and functional tests.
```
Comment (chungg) [inline doc/source/contributing.rst]: s/ugrade/upgrade/

File: doc/source/contributing.rst [hunk 3]
```diff
@@ -33,7 +33,30 @@ When opening a pull-request, make sure that:
...
+When a pull-request is approved by at least two of the members and when
+Travis-CI confirms that all the tests run fine, the patch will be merged.
+
+The Gnocchi project leverages `Pastamaker`_ in order to schedule the merge of
+the different pull-requests. Pastamaker is in charge of making sure that the
+pull-request is up-to-date with respect to the `master` branch and that the
+tests pass. Pull-requests are always merged in a serialized manner in order to
+make sure that no pull-request can break another one.
+
+The `Gnocchi's Pastamaker dashboard`_ shows the current status of the merge
```
Comment (chungg) [inline doc/source/contributing.rst]: s/The//

File: doc/source/contributing.rst (hunk 1 above)
Comment (jd) [reply doc/source/contributing.rst]: Done.

File: doc/source/contributing.rst (hunk 2 above)
Comment (jd) [reply doc/source/contributing.rst]: Done.

File: doc/source/contributing.rst (hunk 3 above)
Comment (jd) [reply doc/source/contributing.rst]: Done.

**Comment by jd** [comment]
Fix typos

**Pull request merged**